Client/src/Components/UserReview/ModalFooter.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from '@material-ui/core'
+import { IDatasetModel } from '../../Models/Datasets/IDatasetModel'
+import React from 'react'
+
+interface IProps {
+  dataset: IDatasetModel,
```

Comment:
this file (and component) should be renamed into DatasetModalApproveFooter, or just delete it and put the buttons in the DatasetModal directly

Reply:
Done